content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -211,14 +211,16 @@ that have the `Topology Affinity` error.
 ### Topology manager policy options
 
 Support for the Topology Manager policy options requires `TopologyManagerPolicyOptions`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled.
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled(enabled by default).
 
 You can toggle groups of options on and off based upon their maturity level using the following feature gates:
-* `TopologyManagerPolicyBetaOptions` default disabled. Enable to show beta-level options. Currently there are no beta-level options.
-* `TopologyManagerPolicyAlphaOptions` default disabled. Enable to show alpha-level options. You will still have to enable each option using the `TopologyManagerPolicyOptions` kubelet option.
+* `TopologyManagerPolicyBetaOptions` default enabled. Enable to show beta-level options.
+* `TopologyManagerPolicyAlphaOptions` default disabled. Enable to show alpha-level options.
+
+You will still have to enable each option using the `TopologyManagerPolicyOptions` kubelet option.
 
 The following policy options exists:
-* `prefer-closest-numa-nodes` (alpha, invisible by default, `TopologyManagerPolicyOptions` and `TopologyManagerPolicyAlphaOptions` feature gates have to be enabled)(1.26 or higher)
+* `prefer-closest-numa-nodes` (beta, visible by default, `TopologyManagerPolicyOptions` and `TopologyManagerPolicyBetaOptions` feature gates have to be enabled)(1.26 or higher)
```

Comment:
```suggestion
* `prefer-closest-numa-nodes` (beta, visible by default; `TopologyManagerPolicyOptions` and `TopologyManagerPolicyBetaOptions` feature gates have to be enabled).  
  The `prefer-closest-numa-nodes` policy option is beta in Kubernetes {{< skew currentVersion >}}.
```
What happens if you had `TopologyManagerPolicyAlphaOptions` enabled and upgrade? Does `TopologyManagerPolicyAlphaOptions` imply `TopologyManagerPolicyBetaOptions`?

(if not, let's call that out - we might also want to make sure that the release notes make this detail clear).

Reply:
It doesn't imply it. But since we have only one option which is also graduating to beta and beta options are enabled by default, it shouldn't require changes to kubelet config unless someone explicitly enabled `alpha` options and disabled `beta` options.